service/src/main/frontend/src/pages/DataDictionary.vue

```diff
@@ -0,0 +1,329 @@
+    >PLACEHOLDER - BANNER</div>
+</template>
+
+<script setup lang="ts">
+import { Ref, UnwrapRef, computed, defineComponent, ref } from 'vue';
+import { QTable, QTableProps, exportFile, useQuasar } from 'quasar';
+import axios from 'axios';
+import * as Formatters from '../functions/formatters';
+
+// Defines Rows and Columns for the Table.
+let rows: QTableProps['rows'] = [];
+const columns: QTableProps['columns'] = [
+  {
+    label: 'Field Name',
+    name: 'fieldName',
+    field: 'fieldName',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 275px; min-width: 275px',
+  },
+  {
+    label: 'Internal FieldName',
+    name: 'internalFieldName',
+    field: 'internalFieldName',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 275px; min-width: 275px',
+  },
+  {
+    label: 'Data Type',
+    name: 'dataType',
+    field: 'dataType',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 100px; min-width: 100px',
+  },
+  {
+    label: 'Index Only',
+    name: 'indexOnly',
+    field: 'indexOnly',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 100px; min-width: 100px',
+  },
+  {
+    label: 'Forward Index',
+    name: 'forwardIndexed',
+    field: 'forwardIndexed',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 100px; min-width: 100px',
+  },
+  {
+    label: 'Reverse Index',
+    name: 'reverseIndexed',
+    field: 'reverseIndexed',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 100px; min-width: 100px',
+  },
+  {
+    label: 'Normalized',
+    name: 'normalized',
+    field: 'normalized',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 100px; min-width: 100px',
+  },
+  {
+    label: 'Types',
+    name: 'Types',
+    field: 'Types',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 100px; min-width: 100px',
+  },
+  {
+    label: 'Tokenized',
+    name: 'tokenized',
+    field: 'tokenized',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 100px; min-width: 100px',
+  },
+  {
+    label: 'Description',
+    name: 'Descriptions',
+    field: 'Descriptions',
+    align: 'center',
+    sortable: true,
+    style: 'max-width: 200px; min-width: 200px',
+  },
+  {
+    label: 'Last Updated',
+    name: 'lastUpdated',
+    field: 'lastUpdated',
+    align: 'left',
+    sortable: true,
+    style: 'max-width: 75px; min-width: 75px',
+  },
+];
+
+// Defines the Table References, loading for axios, search filter, and pagination to sort.
+const table = ref();
+const loading = ref(true);
+const filter = ref('');
+const changeFilter = ref('');
+let paginationFront = ref({
+  rowsPerPage: 23,
+  sortBy: 'fieldName',
+});
+
+// Confirms the table is full as long as the window's size changes.
+window.onresize = () => {
+  paginationFront.value.rowsPerPage = Math.floor(
+    (table.value.$el.clientHeight - 33 - 52 - 28) / 28
+  );
+};
+
+// AXIOS - Loads from REST endpoint.
+axios
+  .get(process.env.ENDPOINT!)
```

Comment:
Can we override this at runtime by setting an environment variable?

More importantly, why do we need to read our own endpoint from an environment variable?  For instance, if we swap this out for '/login' as the endpoint, isn't that an implicit call against our own api/servlet?